example/trace/otlpgrpc/example.go

```diff
@@ -62,7 +65,34 @@ func initTracer() (func(), error) {
 	}, nil
 }
 
+func generateTestSpan(ctx context.Context, tr trace.Tracer, description string) {
+	fmt.Println("starting span...")
+	_, span := tr.Start(ctx, description, trace.WithAttributes(semconv.PeerServiceKey.String("ExampleService")))
+	defer span.End()
+	defer fmt.Println("ending span.")
+
+	time.Sleep(3 * time.Second)
+}
+
+func generateSpansAtFixedRate(ctx context.Context, tr trace.Tracer) {
+	fmt.Println("Generating 1 test span every 10 seconds for 2 hours")
+	totalDuration := 2 * time.Hour
+	endTime := time.Now().Add(totalDuration)
+	ticker := time.NewTicker(10 * time.Second)
+	defer ticker.Stop()
+
+	for tick := range ticker.C {
+		if tick.After(endTime) {
```

Comment:
any reason to set the limit? I think we could just make this run indefinitely

Reply:
No, no technical reason, other than this is not written as a web service and that ADC expires after an hour

Why would we want to keep it indefinite though ?

Reply:
I'd expect it to keep running until I kill it. Hardcoding a limit is kind of unexpected and it means you can only test 1 token refresh

Reply:
Ok, that makes sense to me. I will update it to keep running.

As a side benefit the flag name would perfectly fit the description 😄